src/components/DefineCompany/CooperativeType.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+  <v-form ref="cooperativeTypeForm" id="cooperative-type">
+    <v-row class="py-3" no-gutters>
+      <v-col>
+        <label class="font-weight-bold">Select Type</label>
+      </v-col>
+      <v-col class="ml-n1">
+        <v-select
+          filled
+          persistent-hint
+          id="cooperative-type-input"
+          label="Cooperative Association Type"
+          :items="cooperativeTypes"
+          v-model="cooperativeType"
+          :rules="cooperativeTypeRules"
+        >
+        </v-select>
+      </v-col>
+    </v-row>
+  </v-form>
+</template>
+
+<script lang="ts">
+// Libraries
+import { Component, Emit, Watch, Vue, Prop } from 'vue-property-decorator'
+import { Getter } from 'vuex-class'
+import { CoopType } from '@/enums'
+import { FormType } from '@/interfaces'
+
+@Component({})
+export default class CooperativeType extends Vue {
+  // Refs
+  $refs!: {
+    cooperativeTypeForm: FormType,
```

Comment:
My linter now complains about trailing commas. Yours doesn't?

Reply:
Hmm, maybe not. Will fix!

Reply:
fixed in next commit